src/odemis/gui/cont/fastem_acq.py

```diff
@@ -37,29 +37,33 @@
 
 import wx
 
-from odemis import model, dataio
-from odemis.acq import align, stream, fastem
+from odemis import dataio, model
+from odemis.acq import align, fastem, stream
 from odemis.acq.align import fastem as align_fastem
 from odemis.acq.align.fastem import Calibrations
 from odemis.acq.fastem import estimate_acquisition_time
 from odemis.acq.stream import FastEMOverviewStream
 from odemis.gui import FG_COLOUR_BUTTON
-from odemis.gui.util import get_picture_folder, call_in_wx_main, \
-    wxlimit_invocation
+from odemis.gui.conf.data import get_hw_config
+from odemis.gui.conf.util import process_setting_metadata
+from odemis.gui.util import (call_in_wx_main, get_picture_folder,
+                             wxlimit_invocation)
 from odemis.gui.util.widgets import ProgressiveFutureConnector
 from odemis.util import units
-from odemis.util.dataio import open_acquisition, data_to_static_streams
+from odemis.util.dataio import data_to_static_streams, open_acquisition
 
 
 class FastEMOverviewAcquiController(object):
     """
     Takes care of the overview image acquisition in the FastEM overview tab.
     """
 
-    def __init__(self, tab_data, tab_panel):
+    def __init__(self, tab_data, tab_panel, add_dwell_time_slider=True):
```

Comment:
Please drop this extra option. This class is instantiated only once. Let's just make it always create this slider. That removes a bunch of `if`s.